login-fire-social.html

```diff
@@ -66,24 +67,26 @@
       }
     </style>
 
-    <template is="dom-if" if="[[onlySignOut]]">
+
+
+    <template is="dom-if" if="[[_showOneButton]]">
```

Comment:
Polymer IDE in VS Code was saying it found 2 definitions of the common behavior. This is due to the button importing the common behavior and then social imports the button and the common behavior. This way it doesn't have to deal with a duplicate. Do you know what the best way to deal with this would be? 

It works fine but I don't necessarily like the idea of (for whatever reason) removing common behavior from the button and then that breaks social. Is duplicate import fine?